submissions/EmmanuelIdiaghe/database/transportDB.sql

```diff
@@ -49,4 +49,31 @@ create table accident_status(
     foreign key(passenger_id) references passengers(id)
 );
 
-select * from accident_status;
\ No newline at end of file
+select * from accident_status;
+
+/*********************************************
+** Data Queries
+*********************************************/
+# 1. Total number of passengers who survived
+select count(*) from accident_status where accident_status > 0;
+############# Answer: 342
+
+# 2. Total number of passengers who did not survive
+select count(*) from accident_status where accident_status = 0;
+############# Answer: 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+select passengers.name_of_passenger, passengers.gender from passengers 
+inner join trip_details on passengers.id = trip_details.passenger_id 
+where age < '27' and embarkation_point like "Q%" or embarkation_point like "C%";
+
+# 4. Number of passengers who embarked at Southampton and survived
+select count(*) from accident_status 
+inner join trip_details on accident_status.id = trip_details.passenger_id 
+where accident_status > 0 and embarkation_point like "S%";
+############# Answer: 218
+
+# 5. ID, Name and Total Number of passengers who paid a fare greater than $100 and above the age of 35 had siblings or spouses on board
+select passengers.id, passengers.name_of_passenger, COUNT(*) OVER () AS total_count from passengers 
+inner join trip_details on passengers.id = trip_details.passenger_id 
+where trip_fare > 100 and age > '35' and siblings_or_spouses_aboard > 0;
```

Comment:
Add the result of the count in a comment

Reply:
Done!